Core/wbDefinitionsSF1.pas

```diff
@@ -18650,27 +18650,27 @@ procedure DefineSF1u;
     wbEDID,
     wbFULL,
     wbKeywords,
-    wbString(SNAM),
+    wbString(SNAM, ' Biome Type'),
     wbArray(PNAM, 'Plants', wbFormIDCk('Flora', [FLOR])),
-    wbFormID(RNAM),
+    wbFormIDCk(RNAM, 'Resource Generation Data', [RSGD]),
     wbRStructs('Unknown', 'Unknown', [
       wbString(FNAM),
-      wbUnknown(BNAM),
+      wbUnknown(BNAM),       //Always 01 00 00 00 if it exists
       wbRArray('Unknown', wbStruct(GNAM, 'Unknown', [
         wbFormIDCk('Pack-In', [PKIN]),
         wbUnknown
       ])),
-      wbUnknown(DNAM)
+      wbFloat(DNAM, 'Unknown')
```

Comment:
you know that writing the 'Unknown' part is optional and it will default to that if the parameter is simply not specified?

Reply:
Suspected but wasn't sure.